pkg/utils/time.go

```diff
@@ -76,13 +76,19 @@ func getBootTimeInJiffies() int64 {
 // Boot time functions
 //
 
+// Common clock IDs
+const (
+	CLOCK_MONOTONIC = unix.CLOCK_MONOTONIC // Time since a boot (not including time spent in suspend)
+	CLOCK_BOOTTIME  = unix.CLOCK_BOOTTIME  // Time since a boot (including time spent in suspend)
+)
+
 // GetStartTimeNS returns the system start time in nanoseconds (using CLOCK_MONOTONIC).
```

Comment:
There are many comments in this file that still refer to CLOCK_MONOTONIC, like this one

Reply:
good catch, missed this detail... now can be both, depending on kernel version